crates/egui_demo_lib/src/demo/misc_demo_window.rs

```diff
@@ -264,7 +264,7 @@ impl Widgets {
                 ui.label("This tooltip was created with");
                 ui.monospace(".on_hover_ui(…)");
             });
-            let _ = ui.button("A button you can never press");
+            let _ = ui.button("A button you can press, but that does nothing.");
```

Comment:
Since this demo already has an interactive and a non-interactive tooltip, maybe it could highlight the difference in the text as well, like "This tooltip stays open when the pointer is moved inside it or towards it, because it contains interactive widgets", and "This tooltip closes immediately when the pointer is moved, because it only contains non-interactive widgets" or something like that.